my-profile-middleware-impl/src/main/java/edu/wisc/my/profile/mapper/ContactInformationMapper.java

```diff
@@ -145,7 +146,7 @@ private static JSONObject jsonifyEmergencyContact(ContactInformation eci) {
     JSONObject emergencyContact = new JSONObject();
     emergencyContact.put("EMERGENCY NAME", eci.getPreferredName());
     emergencyContact.put("RELATION", eci.getRelationship());
-    emergencyContact.put("RELATION COMMENT", eci.getComments());
+    emergencyContact.put("RELATION COMMENT", eci.getComments()+" ");
```

Comment:
Should we add in a "if ends in quote then add space"? Baby concern about saving over and over will add more and more spaces.